package.json

```diff
@@ -15,13 +15,15 @@
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
     "@types/react-helmet": "^6.1.2",
+    "@types/react-modal": "^3.12.1",
     "@types/react-router-dom": "^5.1.9",
     "flat": "^5.0.2",
     "node-sass": "4.14.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-helmet": "^6.1.0",
     "react-intl": "^5.20.10",
+    "react-modal": "^3.14.3",
```

Comment:
Is there any reason for adding react modal instead of using the one provided by meterial ui?

Reply:
you are right, there is no need of using react-modal, I will remove this dependency and use the MUI Modal